src/Relude/Foldable/Fold.hs

```diff
@@ -230,12 +227,11 @@ Just True
 >>> anyM (readMaybe >=> pure . even) ["aba", "10"]
 Nothing
 -}
-anyM :: (Foldable f, Monad m) => (a -> m Bool) -> f a -> m Bool
+anyM :: forall f m a . (Foldable f, Monad m) => (a -> m Bool) -> f a -> m Bool
 anyM p = foldr go (pure False)
   where
-    go x acc = do
-        q <- p x
-        if q then pure True else acc
+    go :: a -> m Bool -> m Bool
+    go x = ifM (p x) (pure True)
 {-# INLINEABLE anyM #-}
```

Comment:
Since these functions become very short after this refactoring, we can replace all 4 `INLINEABLE` with `INLINE` for `andM`, `orM`, `anyM` and `allM`.

Reply:
Good catch! Done.